src/checkbox/index.js

```diff
@@ -20,7 +20,9 @@ const Checkbox = ({
     <label className={classNames}>
       <input
         type="checkbox"
+        name={label} // when name prop isn't passed, name sets to same value as label prop
         onClick={() => onClick(label, label)}
+        id={checked ? 'checked' : undefined}
```

Comment:
Why do you need this conditional statement ?

Reply:
Fixed in #52